alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/java/org/alfresco/transformer/AIOController.java

```diff
@@ -135,11 +127,7 @@ protected void executeTransformCommand(File sourceFile, File targetFile)
         @RequestParam(SOURCE_MIMETYPE) String sourceMimetype,
         @RequestParam(TARGET_MIMETYPE) String targetMimetype,
         @RequestParam Map<String, String> requestParameters,
-        @RequestParam (value = TEST_DELAY, required = false) Long testDelay,
-
-        // The TRANSFORM_NAME_PROPERTY param allows ACS legacy transformers to specify which transform to use,
-        // It can be removed once legacy transformers are removed from ACS.

```

Comment:
Is this comment line true ? Do we actually need to wait for legacy transformers to actually be removed from ACS ... ?

... or, as per this PR, remove now since we are adding T-Config for all remaining transforms (~ legacy) that are supported by the current T-Engines.

In other words, I want to ensure that even if we remove this code, release a new version of T-Engines (eg. 2.2.2) *and* deploy with existing ACS 6.2.1 then it will continue to work via local transforms as per new T-Config.

cc @eknizat @PunchedChimera

Reply:
@KikoUK What's the plan to resolve above comment ? 

If we decide to leave above code in (until we know new transforms are working, eg. via automated tests) then we should also add an INFO log message to indicate if Repo is still unexpectedly going via this code path.